README.md

```diff
@@ -86,6 +128,30 @@ is present.
 
 ### Elided Imports
 
+The loader looks for `has pragmas`, which are strings that contain a call to has for a specific feature, and 
```

Comment:
Feeling like @dylans, is it `has pragmas` or *has pragmas* because it isn't really code that you are indicating.